### PR TITLE
Add support for ConfigParser's "strict" keyword argument.

### PR DESCRIPTION
**Pull request by jsivak**
Howdy, 

Anyconfig does not currently support passing the "strict" keyword argument to the ConfigParser object.

This pull requests adds that support.

Thanks

### CODE REVIEW

File: src/anyconfig/backend/ini.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ class Parser(base.Parser, base.FromStreamLoaderMixin,
     _type: str = 'ini'
     _extensions: typing.List[str] = ['ini']
     _load_opts: typing.List[str] = [
-        'defaults', 'dict_type', 'allow_no_value', 'filename', 'ac_parse_value'
+        'defaults', 'dict_type', 'allow_no_value', 'filename', 'ac_parse_value', 'strict'
```
Comment (ssato) [inline src/anyconfig/backend/ini.py:189]: > __[Flake8]__ line too long (89 > 79 characters) ([view](https://sider.review/gh/repos/4830526/pulls/150#issue-8324193))
>
> <details>
>
> | Rule |
> | ---- |
> | `E501` |
>
> You can [close this issue](https://sider.review/gh/repos/4830526/pulls/150#issue-8324193) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

This needs to be fixed.

**Comment by ssato** [comment]
Thanks for your request. Almost LGTM but I prefer all checks to be passed.

Could you please fix the line Sider pointed out and I commented.

**Comment by jsivak** [comment]
I updated the code with a shorter line.

**Comment by jsivak** [comment]
I mistyped the last commit; it should read "Remove trailing space"

**Comment by ssato** [comment]
LGTM and merged. Thanks a lot!

**Pull request merged**